image_substitutors_test.go

```diff
@@ -66,6 +66,14 @@ func TestPrependHubRegistrySubstitutor(t *testing.T) {
 
 			require.Equalf(t, "my-registry/org/user/foo:latest", img, "expected my-registry/org/foo:latest, got %s", img)
 		})
+
+		t.Run("registry with port", func(t *testing.T) {
```

Comment:
NIT: we are moving the subtest to this format (without spaces). Existing tests are untouched, yet, but we encourage new ones to not contain them.
```suggestion
		t.Run("registry-with-port", func(t *testing.T) {
```